server/src/server.js

```diff
@@ -161,8 +165,8 @@ app.put("/data/:id/:domain", function (req, res) {
       return shot.update();
     }
     return null;
-  }).then(() => {
-    simpleResponse(res, "Saved", 200);
+  }).then((ok) => {
+    simpleResponse(res, JSON.stringify(shot.asJson()), 200);
```

Comment:
This feels a little like a lie: we're sending back the entire shot as JSON, implying this is the new representation, but the client is only looking at the clip URLs.